File: src/main/php/Easy/StudentMinimumMovesToSeat.php
<?php

declare(strict_types=1);

namespace Src\Easy;

class StudentMinimumMovesToSeat
{
    /**
     * @param  int[]  $seats
     * @param  int[]  $students
     */
    public function solve(array $seats, array $students): int
    {
        sort($seats);
        sort($students);
        $moves = 0;
        foreach ($seats as $i => $seat) {
            $moves += abs($seat - $students[$i]);
        }

        return $moves;
    }
}

File: src/main/cpp/easy/student_minimum_moves_to_seat.cpp
#include "student_minimum_moves_to_seat.h"

int studentMinimumMovesToSeat(std::vector<int>& seats, std::vector<int> & students) {
	int moves = 0;
	sort(seats.begin(), seats.end());
	sort(students.begin(), students.end());
	for(size_t i = 0; i < seats.size(); i++) {
		moves += std::abs(seats[i] - students[i]);
	}
	return moves;
}